lib/importproject.cpp

```diff
@@ -715,37 +719,85 @@ bool ImportProject::importVcxproj(const std::string &filename, std::map<std::str
                         }
                     }
                 }
-            } else {
-                for (const tinyxml2::XMLElement *e = node->FirstChildElement(); e; e = e->NextSiblingElement()) {
+            }
```

Comment:
Please try to avoid whitespace changes.

Reply:
Seems like my Visual Studio instance messed up the formatting _a lot_. I manually formatted my few changes, I hope this is more readable now.